README.md

```diff
@@ -246,7 +249,17 @@ make -f MakefileExternal LIBTRIXI_PREFIX=path/to/libtrixi/prefix
 ```
 to build `simple_trixi_controller_f`.
 
-Note: On Linux and FreeBSD systems (i.e., *not* on macOS or Windows), Julia may internally
+#### CMake
+
+A CMake module for the discovery of an installed libtrixi library is provided with
+[`cmake/FindLibTrixi.cmake`](cmake/FindLibTrixi.cmake). Upon invocation of
+`find_package(LibTrixi)`, the CMake variable `LIBTRIXI_PREFIX` must be set to
+`<install_directory>`. An example `CMakeLists.txt` can be found in
```

Comment:
```suggestion
[`cmake/FindLibTrixi.cmake`](cmake/FindLibTrixi.cmake). Before calling
`find_package(LibTrixi)`, the CMake variable `LIBTRIXI_PREFIX` must be set to
`<install_directory>`. An example `CMakeLists.txt` can be found in
```
Maybe this makes it clearer? To me, "upon invocation ..." sounds a little like "when it is invoked, it will do this and that", whereas you want to express that this needs to be defined beforehand (IIRC).